helmfile.d/values/falco/falco-common.yaml.gotmpl

```diff
@@ -161,6 +161,8 @@ customRules:
         - registry.k8s.io/sig-storage/csi-provisioner
         - registry.k8s.io/sig-storage/csi-resizer
         - registry.k8s.io/sig-storage/csi-snapshotter
+        - registry.k8s.io/cluster-api-azure/cluster-api-azure-controller
+        - mcr.microsoft.com/k8s/azureserviceoperator
```

Comment:
Please keep this list sorted.

Reply:
I have now run the list trough `sort` :slightly_smiling_face: